taskrun/ClusterTask.py

```diff
@@ -214,7 +233,7 @@ def _build_command(self):
       cmd.append(re.sub('"', '\\"', self._command))
       return ' '.join(cmd)
     elif self._mode == 'slurm':
-      cmd = ['srun', '-J', self.name]
+      cmd = ['srun', '-vvvv', '-J', self.name]
```

Comment:
Now that we are added functionality, there are a few cases we should worry about here. 
1. The user doesn't specify any files to capture. In this case ClusterTask should behave like ProcessTask and capture the stdout and stderr into member variables. In this case we wouldn't want -vvvv.
2. The user specifies std{out,err} and logfile. In this case we want -vvvv and execution would yield 2 or 3 files (depending if out and err are combined or not).
3. The user only specifies logfile, but not std{out/err}. This is a weird case. Should it be supported? If so, we would push stdout, stderr, and srun out to the logfile.

Reply:
Right now the job std{out,err} are piped to /dev/null if you don't ask for them. In order for ClusterTask to be a drop-in replacement, we should instead save them to a temporary file and copy them to the stdout and stderr member variables. Should I do that here, or make an issue and punt on it?

Reply:
Punt on it for now.